src/controllers/confirmation.controller.ts

```diff
@@ -79,13 +79,12 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
         logger.error("No reason_code in api response" + apiResponseData);
         return next(new Error("No reason_code in api response"));
       }
-      if (cannotUseReason !== eligibilityReasonCode.COMPANY_IS_ELIGIBLE) {
-        return res.render(Templates.NOT_ELIGIBLE,
-          {
-            cannotUseReason,
-            companyName: companyProfile.companyName,
-          });
-      }
+
```

Comment:
Removed unneeded conditional - COMPANY_IS_ELIGIBLE is never returned by API in the case of a 400 response. Have tested the app and all working without it.